comp/otelcol/logsagentpipeline/logsagentpipelineimpl/agent_test.go

```diff
@@ -185,8 +186,10 @@ func TestBuildEndpoints(t *testing.T) {
 		configComponent.MockModule(),
 		fx.Provide(func() log.Component { return logmock.New(t) }),
 	))
+	deps.Config.Set("logs_config.force_use_http", true, pkgconfigmodel.SourceDefault)
```

Comment:
Why did this change sneak in? Looks unrelated?

Reply:
This test was failing without this change because TCP doesn't support any headers (including the intake origin header). We also use this setting in both DDOT and OSS Collector